src/utils/generate-gateway-key.ts

```diff
@@ -1,9 +1,9 @@
 import { randomBytes } from 'crypto';
-import { runAndHandleErrors } from './utils/cli';
+import { runAndHandleErrors } from './cli';
 
 const main = async () => {
-  const httpGatewayKey = randomBytes(48).toString('hex');
-  const signedHttpGatewayKey = randomBytes(48).toString('hex');
+  const httpGatewayKey = randomBytes(32).toString('hex');
+  const signedHttpGatewayKey = randomBytes(32).toString('hex');
```

Comment:
I'm guessing this change belongs to another issue, right?

Reply:
its just a QOL change, the generated keys were too big